.github/workflows/codeql-analysis.yml

```diff
@@ -13,12 +13,6 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "main" ]
-  pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [ "main" ]
-  schedule:
-    - cron: '31 14 * * 1'
```

Comment:
Do we want to remove the schedule?

Reply:
I don't know why to keep it if it will run on every push/change.
But if there is a reason I'm okay with keeping it.

Reply:
I would say because of security reasons, new security vulnerabilities can be found even with the dependencies fixed. This will flag those out, wouldn't it?

Reply:
I put it back ✅